eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/SubSnpNoHgvs.java

```diff
@@ -86,12 +86,24 @@ public SubSnpNoHgvs(Long ssId, Long rsId, String reference, String alleles, Stri
         this.ssCreateTime = ssCreateTime;
         this.rsCreateTime = rsCreateTime;
         this.taxonomyId = taxonomyId;
+        this.anyOrientationUnknown = (subsnpOrientation.getValue() * snpOrientation.getValue()
+                * contigOrientation.getValue()) == 0;
         Orientation allelesOrientation = Orientation.getOrientation(
-                subsnpOrientation.getValue() * snpOrientation.getValue() * contigOrientation.getValue());
+                composeOrientationAssumingForwardIfUnknown(subsnpOrientation, snpOrientation, contigOrientation));
         this.dbSnpVariantAlleles = new DbsnpVariantAlleles(reference, alleles, contigOrientation, allelesOrientation,
                                                            this.dbsnpVariantType);
     }
 
+    private int composeOrientationAssumingForwardIfUnknown(Orientation subsnpOrientation, Orientation snpOrientation,
```

Comment:
I think the name is somewhat misleading because it could mean that as soon as one of the flags is unknown we return `Orientation.UNKNOWN`. It think it would be better named as `composeOrientation` and clarify it via method documentation.

Also, is there any benefit on it not returning the result from the `Orientation.getOrientation()` method?